src/Makefile.in

```diff
@@ -984,10 +984,19 @@ peflags:
 	@echo Securing executables.
 	$(PEFLAGS) $(PROJ)
 
+###############################################################################
+# To use it, run
+# $ make -s windows-tiny-link-to-john _BEST_SIMD=\"\\\"john-avx2.exe\\\"\"
+###############################################################################
+.PHONY: windows-tiny-link-to-john
+windows-tiny-link-to-john:
+	@echo Build a tiny john.exe that calls the best SIMD binary.
+	$(CC) -DMAIN_NAME=$(_BEST_SIMD) symlink.c -o ../run/john.exe
+	$(STRIP) ../run/john.exe
+
 .PHONY: windows-package
 windows-package: default
-	@echo Deploying for Windows.
-	$(MAKE) peflags
```

Comment:
Do we remove `$(MAKE) peflags` here because we rely on the check for "Windows_NT" (IIRC) in the default build (and need that check to work anyway, because not everyone will use a package)?

Reply:
> Do we remove $(MAKE) peflags here because we rely on the check for "Windows_NT" 

Yes.